dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,212 @@
+  instagram: Onze evenementen en UPdates
+  bulletPoints:
+    - title: Dansavonden
+      description: Elke week komen we samen voor drankjes, vaak is er zelfs een DJ om die voetjes op te bewegen!
+      image: events-dancenights
+    - title: Feesten
+      description: Disco, hard rock, Oktoberfest, Halloween en nog veel meer... Er is altijd tijd voor een feestje.
+      image: events-parties
+    - title: Drag optredens
+      description: Ons eigen drag huis HAUS of 4D geeft elke twee maanden geweldige shows
+      image: events-dragshow
+    - title: Evenementen
+      description: Karaoke, proeverijen, vinyl nachten of een talentenshow, er is veel te beleven op een MIXUP avond.
+      image: events-silentdisco
+  highlights:
+    title: Onze Hoogtepunten
+    events:
+      - title: MIXUP Oorsprong
+        description: Het allereerste feestje onder de MIXUP banner was MIXUP Origin’s, het openingsfeest! Miss Abby OMG gaf ons een prachtig optreden, het feest was aan!
+        image: highlight-abbyomg
+      - title: Zomerfeest
+        description: Heet weer en nog hetere mensen! DJ Maarten verzorgde tropische tunes terwijl we zomercocktails aan het nippen waren en afkoelde in een daadwerkelijk zwembad!
+        image: highlight-summer-party
+</i18n>
+
+<script setup>
+useHead({
+  bodyAttrs: {
+    class: 'mixup-colors',
+  },
+})
+
+const { t, tt } = useT()
+
+const { data: openingHours } = await useAsyncData(() => queryContent('opening_hours').findOne())
+const barOpeningHours = openingHours.value.events.find((event) => event.day.en === 'Saturday')
+
+const { image: imageOverviews } = useDynamicImages(import.meta.glob('~/assets/images/photos/mixup/*', { eager: true }))
+
+const instagramChannelsMixup = [
+  {
+    name: 'MIXUP',
+    widgetId: '219aedf13a4355fe95c6e27ed1f7386c',
+    instagram: 'mixupdelft',
+  },
+]
+</script>
+
+<template>
+  <LayoutSmallHeader triangleClass="border-brand-450">
+    {{ t('title') }}
+  </LayoutSmallHeader>
+
+  <section>
+    <PagesMixupLinkedEvents />
+  </section>
+
+  <section>
+    <div class="bg-brand-900 text-white text-lg mx-auto pt-12 pb-24 md:flex">
+      <div class="flex-1 px-4 lg:pr-32">
+        <div class="space-y-4">
+          <ElementsParagraphedText :paragraphs="t('intro')" class="md:text-xl md:leading-relaxed space-y-4" />
+          <p
+            v-if="barOpeningHours.announcement"
+            class="mt-3 mb-4 text-brand-900 bg-brand-450"
+            v-text="tt(barOpeningHours.announcement)"
+          />
+          <div
+            class="flex-1 rounded-lg shadow-xl bg-brand-450 md:text-xl p-4 relative w-full z-20 md:w-auto md:inline-flex items-center"
+          >
+            <div class="pt-6 md:pt-0 md:pl-4 space-y-2">
+              <p>{{ t('invite.announcement') }}</p>
+              <p class="font-bold">{{ t('invite.time', [barOpeningHours.start_time]) }}</p>
+            </div>
+          </div>
+          <div class="flex flex-1 space-x-4">
```

Comment:
In terms of hierarchy and stuff it makes sense to have a single/main call to action (a secondary one is fine). Maybe pick one and change its colors.

Maybe it's an idea to leave out the membership button, and make a separate box here with the bar buddy button + the little description that's currently in the intro text

Reply:
It was a request from Colin to have the membership button there. I'll discuss with him